client/packages/invoices/src/InboundShipment/DetailView/Toolbar.tsx

```diff
@@ -138,8 +119,9 @@ export const Toolbar: FC<{
           </Box>
           <DropdownMenu label={t('label.actions')}>
             <DropdownMenuItem
+              disabled={shipment?.status === InvoiceNodeStatus.New}
```

Comment:
Doesn't look like InboundShipmentLines have a stock line until they are changed to delivered status... do you think this requires a tooltip explanation or just disabling is fine?

Reply:
I think our current strategy with UI/UX is to avoid disabling, and to show a warning, would prefer this tbh.